app/helpers/sufia/sufia_helper_behavior.rb

```diff
@@ -121,6 +121,11 @@ def iconify_auto_link(text, showLink = true)
       end
     end
 
+    def iconify_link(text, showLink = true)
```

Comment:
sure i'll leave out the 2nd arg and someone can add it back in in the future if there's another use case. happy to fix the camel case var in the method above while i'm in there.